src/css/base.scss

```diff
@@ -1901,10 +1933,6 @@ button.anchor {
     padding-left: 220px;
 }
 
-.h-32 {
-    height: 32px;
```

Comment:
Is this intentional?

Reply:
yes. same name se 2 class bni hui thi base me